lib/ui/pages/coursedetail/screen/album_page.dart

```diff
@@ -0,0 +1,115 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_app/src/store/picture_storage.dart';
+import 'package:photo_view/photo_view.dart';
+import 'dart:io';
+
+class AlbumPage extends StatefulWidget {
+  final String courseId;
+
+  const AlbumPage({Key? key, required this.courseId}) : super(key: key);
+
+  @override
+  State<AlbumPage> createState() => _AlbumPageState();
+}
+
+class _AlbumPageState extends State<AlbumPage> {
+  bool isLoading = true;
+
+  late int pictureNum;
+  late List<Picture> pictures = [];
+
+  dynamic picturesInfo;
+
+  late double screenWidth;
+  late double screenHeight;
+
+  @override
+  void initState() {
+    super.initState();
+    Future.delayed(Duration.zero, () {
+      useCourseIdToGetPicturePaths();
+    });
+  }
+
+  void useCourseIdToGetPicturePaths() async {
+    picturesInfo = await PictureStorage.getCoursePicture(widget.courseId);
+    isLoading = false;
+    pictureNum = picturesInfo.length;
+    for (int i = 0; i < pictureNum; i++) {
+      int infoId = picturesInfo[i]['_id'];
+      String infoLabel = picturesInfo[i]['label'];
+      String infoNote = picturesInfo[i]['note'];
+      String infoPath = picturesInfo[i]['picturePath'];
+
+      pictures.add(Picture(infoId, infoLabel, infoNote, infoPath));
+    }
```

Comment:
This can be simplified.

```suggestion
    for (final pictureInfo in pictureInfos) {
      int infoId = pictureInfo['_id'];
      String infoLabel = pictureInfo['label'];
      String infoNote = pictureInfo['note'];
      String infoPath = pictureInfo['picturePath'];

      pictures.add(Picture(infoId, infoLabel, infoNote, infoPath));
    }
```